workflow/rules/plot.smk

```diff
@@ -0,0 +1,20 @@
+rule interactive_plots:
+    input:
+        tsv="results/tables/diffexp/{model}.transcripts.diffexp.tsv",
+    output:
+        volcano_plot=directory(
+            report(
+                "results/plots/interactive/volcano/{model}/",
+                caption="../report/plot-volcano.rst",
+                category="QC",
```

Comment:
As you pointed out in the PR message, this is probably not the best category for it. As it is an actual result and can generate multiple plots across multiple models, it could even go into its own `Volcano plot` category.

Reply:
Or a general "interactive plots" category with a volcano subcategory. There are a lot of options, we just have to pick one ;)